transport/src/main/java/io/camunda/zeebe/transport/stream/impl/RemoteStreamTransport.java

```diff
@@ -72,34 +70,34 @@ public void removeAll(final MemberId member) {
 
   private byte[] onAdd(final MemberId sender, final AddStreamRequest request) {
     requestHandler.add(sender, request);
-    return EMPTY_RESPONSE;
+    return EMPTY_REQUEST;
   }
 
   private byte[] onRemove(final MemberId sender, final RemoveStreamRequest request) {
     requestHandler.remove(sender, request);
-    return EMPTY_RESPONSE;
+    return EMPTY_REQUEST;
   }
 
   private byte[] onRemoveAll(final MemberId sender, final byte[] ignored) {
     requestHandler.removeAll(sender);
-    return EMPTY_RESPONSE;
+    return EMPTY_REQUEST;
   }
 
   public void recreateStreams(final MemberId receiver) {
     try {
       sendRestartStreamsCommand(receiver);
-      LOG.trace("Tried to restart streams with member: {}", receiver);
+      LOG.debug("Tried to restart streams with member: {}", receiver);
     } catch (final Exception e) {
-      LOG.debug("Failed to restart streams with member: {}", receiver);
+      LOG.warn("Failed to restart streams with member: %s".formatted(receiver), e);
```

Comment:
:x: You can keep using the `{}` format. All SLF4J calls accept an error as the last argument - that's not formatted into the log message, and instead is added as context info.

So you can do:

```java
LOG.warn("Failed to restart streams with member: {}", receiver, e);
```